test_util/src/lib.rs

```diff
@@ -880,7 +880,7 @@ impl CheckOutputIntegrationTest {
     reader.read_to_string(&mut actual).unwrap();
 
     let status = process.wait().expect("failed to finish process");
-    let exit_code = status.code().unwrap();
+    let exit_code = status.code().unwrap_or(0);
```

Comment:
I don't like this. It can potentially swallow errors in tests.

Reply:
Then why did this occur, only on Ubuntu? https://github.com/denoland/deno/runs/1306229988#step:14:3856

Reply:
I don't know. The status code is a `None` value when the process was terminated by a signal. See https://doc.rust-lang.org/std/process/struct.ExitStatus.html

It is not safe to assume that it exited cleanly (code 0) on a `None` code.

Reply:
Ok, I have removed it... see what happens this time with CI.

Reply:
Same problem: https://github.com/denoland/deno/pull/8123/checks?check_run_id=1306322095#step:14:3858

It isn't occurring locally on my mac, and I don't know how the test case could be invalid.